src/gui/transferlistwidget.cpp

```diff
@@ -393,6 +393,15 @@ void TransferListWidget::copySelectedNames() const
     qApp->clipboard()->setText(torrent_names.join("\n"));
 }
 
+void TransferListWidget::copySelectedHashes() const
+{
+    QStringList torrent_hashes;
```

Comment:
the coding guidelines is added after this file was coded and we haven't cleaned up this file yet, so just ignore the inconsistency.